src/lib/meta/meta_sb.hpp

```diff
@@ -91,6 +91,7 @@ struct MetaSubRegInfo {
     std::set< uint64_t > meta_bids; // meta blk id
     meta_blk_found_cb_t cb{nullptr};
     meta_blk_recover_comp_cb_t comp_cb{nullptr};
+    bool has_deps;
```

Comment:
Needs to be defaulted to false

Reply:
the default value of bool is false, no need to set it explicitly

Reply:
Isn't it depending on the compiler? Why should we stop a good habit of always initializing the var before accessing it, not to mention any complaining code analysis's tool that would report issues on it.